src/components/AutoCompleteSuggestions/AutoCompleteSuggestionsPortal/index.native.tsx

```diff
@@ -0,0 +1,34 @@
+import {Portal} from '@gorhom/portal';
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import BaseAutoCompleteSuggestions from '@components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions';
+import useStyleUtils from '@hooks/useStyleUtils';
+import getBottomSuggestionPadding from './getBottomSuggestionPadding';
+import type {AutoCompleteSuggestionsPortalProps} from './types';
+
+function AutoCompleteSuggestionsPortal<TSuggestion>({left = 0, width = 0, bottom = 0, ...props}: AutoCompleteSuggestionsPortalProps<TSuggestion>) {
+    const StyleUtils = useStyleUtils();
+    const styles = useMemo(() => StyleUtils.getBaseAutoCompleteSuggestionContainerStyle({left, width, bottom: bottom + getBottomSuggestionPadding()}), [StyleUtils, left, width, bottom]);
+    // const styles = useMemo(() => StyleUtils.getBaseAutoCompleteSuggestionContainerStyle({left, width, bottom: bottom + 0}), [StyleUtils, left, width, bottom]);
```

Comment:
Why is this commented out?

Reply:
I overlooked that comment. I'll remove it right away. Thank you!